### ISSUE TITLE
Multibyte characters break when unserialising

### ISSUE THREAD

**Issue by roborourke** [humanmade/Cavalcade#85]
Jobs containing args that contain multibyte unicode characters such as emoji can't be run as after they're serialised they do not unserialise correctly.

**Comment by rmccue** [comment]
I believe this is an issue with the table and/or column encoding not supporting utf8mb4?

**Comment by roborourke** [comment]
Yeah doesn't look like the db table creation scripts force utf8mb4.

**Comment by roborourke** [comment]
Time for an `upgrade_database_3()` function I think.

**Comment by peterwilsoncc** [comment]
I’ve started working on a third version of the DB scheme in #86 to account for the new preflight filters. 

I might split it out the DB upgrade in to its own PR and incorporate this to avoid multiple upgrade steps. 

WordPress core does some voodoo to support multibyte characters in older versions of MySQL. 

I figure we can leave out the voodoo and limit the check to make sure the server supports multibyte characters. Without such support, Cavalcade doesn’t support the characters in arguments
